iroh-bytes/src/protocol/range_spec.rs

```diff
@@ -1,32 +1,47 @@
-//! Specifications for individual ranges within a blob, and for ranges for a
-//! collection and its children.
+//! Specifications for ranges selection in blobs and collections.
+//!
+//! The [`RangeSpec`] allows specifying which BAO chunks inside a single blob should be
+//! selected.
+//!
+//! The [`RangeSpecSeq`] builds on top of this to select blob chunks in an entire
+//! collection.
```

Comment:
"...and its childen" ?

Reply:
thinking about this some more I think we should really leave the notion of a collection out of this, and explain how this is used to encode collections and its children in a different place.